Library/Formula/terraform.rb

```diff
@@ -145,14 +63,10 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
-
+      system "go test $(go list ./... | grep -v /vendor/)"
       mkdir "bin"
       arch = MacOS.prefer_64_bit? ? "amd64" : "386"
-      system "gox", "-arch", arch,
-        "-os", "darwin",
-        "-output", "bin/terraform-{{.Dir}}",
-        "./..."
+      system "gox -arch #{arch} -os darwin -output bin/terraform-{{.Dir}} $(go list ./... | grep -v /vendor/)"
```

Comment:
We might want to cache the output of `go list./... | grep -v /vendor/` since we’re doing it twice.